back/src/main.ts

```diff
@@ -2,10 +2,43 @@ import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
 import { PrismaService } from './prisma/prisma.service';
 import { SwaggerModule, DocumentBuilder } from '@nestjs/swagger';
-import { ValidationPipe } from '@nestjs/common';
+import { CallHandler, ExecutionContext, Injectable, NestInterceptor, ValidationPipe } from '@nestjs/common';
+import {RequestLogger, RequestLoggerOptions} from 'json-logger-service';
+import { tap } from 'rxjs';
+
+@Injectable()
```

Comment:
There are a mix of spaces and tabs for indentation. run prettier or something

Reply:
ran prettier